solr/core/src/test/org/apache/solr/filestore/TestDistribPackageStore.java

```diff
@@ -39,15 +40,11 @@
 import org.apache.zookeeper.server.ByteBufferInputStream;
 import org.junit.After;
 import org.junit.Before;
-
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.ByteBuffer;
 import java.nio.file.Paths;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
please configure your IDE to not do this

Reply:
Ok, I can fix this one.